### ISSUE TITLE
migration command

### ISSUE THREAD

**Issue by ILLISIS** [Hornwitser/eternity_commands#2]
This sounds doable?

current process:

1) stop the instance
2) rename latest save to instance name-<date>
3) download the latest save
4) reassign the instance to XX host
5) upload the save

expected behaviour
ec migrate < instance > < host >

thanks man

**Comment by ILLISIS** [comment]
Could even do 

ec migrate --instances < instance >,< instance >,< instance > --host < host >

**Comment by Hornwitser** [comment]
Implemented in 519c772

**Sample usage**

Migrate Copper Smelting 01 to eu1-node1:
```
clusterioctl ec migrate --instances "Copper Smelting 01" --to-hosts eu1-node1
```

Migrate multiple instances to one host
```
clusterioctl ec migrate --instances "Copper Smelting 01" "Copper Smelting 02" "Copper Smelting 03" --to-hosts eu1-node1
```

Migrate all instances currently on eu1-node1 to us2-node1
```
clusterioctl ec migrate --from-hosts eu1-node1 --to-hosts us2-node1
```

Migrate all instance on multiple hosts, with a round robin assignment to multiple targets.
```
clusterioctl ec migrate --from-hosts us1-node1 us2-node2 --to-hosts eu6-node1 eu6-node2
```
